docs/pages/admin-guides/access-controls/access-request-plugins/ssh-approval-datadog.mdx

```diff
@@ -0,0 +1,381 @@
+it needs to communicate with your Teleport cluster and the Datadog API. In this step,
+you will configure the Datadog Incident Management plugin to use these credentials.
+You will also configure the plugin to notify the desired recipient when it receives
+an Access Request update.
+
+### Create a configuration file
+
+<Tabs>
+<TabItem label="Executable or Docker">
+The Teleport Datadog Incident Management plugin uses a configuration file in TOML
+format. Generate a boilerplate configuration by running the following command
+(the plugin will not run unless the config file is in `/etc/teleport-datadog.toml`):
+
+```code
+$ teleport-datadog configure | sudo tee /etc/teleport-datadog.toml > /dev/null
+```
+
+This should result in a configuration file like the one below:
+
+```toml
+(!examples/resources/plugins/teleport-datadog.toml!)
+```
+</TabItem>
+<TabItem label="Helm Chart">
+The Datadog Helm Chart uses a YAML values file to configure the plugin.
+On your local workstation, create a file called `teleport-datadog-helm.yaml`
+based on the following example:
+
+```toml
+(!examples/resources/plugins/teleport-datadog-helm.yaml!)
+```
+
+</TabItem>
+</Tabs>
+
+### Edit the configuration file
+
+Open the configuration file created for the Teleport Datadog Incident Management
+plugin and update the following fields:
+
+**`[teleport]`**
+
+The Slack plugin uses this section to connect to your Teleport cluster:
+
+(!docs/pages/includes/plugins/config-toml-teleport.mdx!)
+
+(!docs/pages/includes/plugins/refresh-plugin-identity.mdx!)
+
+**`[datadog]`**
+
+`api_endpoint`: Specifies the desired Datadog API endpoint. See
+[`https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site`](https://docs.datadoghq.com/getting_started/site/#access-the-datadog-site),
+for available Datadog regions.
+
+`api_key`: Open [`https://app.datadoghq.com/organization-settings/api-keys`](https://app.datadoghq.com/organization-settings/api-keys),
+find the "Teleport Access" API key you created earlier and copy the key into this
+field.
+
+`application_key`: Open [`https://app.datadoghq.com/organization-settings/application-keys`](https://app.datadoghq.com/organization-settings/application-keys),
+find the "Teleport Access" Application key you created for your service account
+and copy the key into this field.
+
+`severity`: Specifies the Datadog incident severity.
+
+**`[role_to_recipients]`**
+
+The `role_to_recipients` map configures the recipients that the Datadog Incident Management
+plugin will notify when a user requests access to a specific role. When the plugin
+receives an Access Request from the Auth Service, it will look up the role being
+requested and identify the Datadog users or teams to notify.
+
+<Tabs>
+<TabItem label="Executable or Docker">
+Here is an example of a `role_to_recipients` map:
+
+```toml
+[role_to_recipients]
+"*" = ["admin-team"]
+"dev" = ["dev-team", "admin-team"]
+"dba" = ["user@example.com"]
```

Comment:
Role to recipients is deprecated in favor of notification routing rules. Let's not use this anymore.